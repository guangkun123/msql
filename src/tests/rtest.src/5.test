# Do some lookups (one wildcard,one keyed, one non-keyed, and one regexp)

select phone from test where user = 'Bambi'\p\g

select * from test2 \p\g

select * from test where phone = '(075) 951412' \p\g

select * from test where user like '%am%' \p\g
