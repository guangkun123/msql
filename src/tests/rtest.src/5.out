
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

select phone from test where user = 'Bambi'
[continue]
    -> 
Query OK.

1 rows matched.

 +----------------------+
 | phone                |
 +----------------------+
 | (075) 951450         |
 +----------------------+



mSQL >     -> 
Query buffer
------------
select * from test2 
[continue]
    -> 
Query OK.

3 rows matched.

 +------------+--------------------------------+
 | user       | fullname                       |
 +------------+--------------------------------+
 | paulp      | Paul Pyyvaara                  |
 | Bambi      | David J. Hughes                |
 | fred       | Fred O'Niell                   |
 +------------+--------------------------------+



mSQL >     -> 
Query buffer
------------
select * from test where phone = '(075) 951412' 
[continue]
    -> 
Query OK.

1 rows matched.

 +------------+----------+----------------------+
 | user       | age      | phone                |
 +------------+----------+----------------------+
 | paulp      | 25       | (075) 951412         |
 +------------+----------+----------------------+



mSQL >     -> 
Query buffer
------------
select * from test where user like '%am%' 
[continue]
    -> 
Query OK.

1 rows matched.

 +------------+----------+----------------------+
 | user       | age      | phone                |
 +------------+----------+----------------------+
 | Bambi      | 26       | (075) 951450         |
 +------------+----------+----------------------+



mSQL > 
Bye!

